docs/bootnode.md

```diff
@@ -0,0 +1,25 @@
+# Bootnode
+
+The swap program uses a p2p network to discover offer-makers to do a swap with, and also the run the actual swap protocol. A node must know addresses of nodes already in the network to join the network. These nodes are often publicly posted and referred to as bootnodes. Bootnodes act as an entry-point into the p2p network. 
+
+This repo comes with a `bootnode` program that runs only the p2p components of a swap node, and thus can be used as a lightweight bootnode. 
+
+## Requirements
+- go 1.20+ (see [build instructions](./build.md) to download Go.)
```

Comment:
Maybe we should say "See ... " for install help and minimum supported version?  It is something we have to update twice per year, so I'm trying to brainstorm ways that we could minimize the number of places that require updating. Obviously, `grep ^go go.mod` is the authoritative source for the minimum version.